code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -270,7 +270,7 @@
 	caliber = "a556"
 	matter = list(DEFAULT_WALL_MATERIAL = 1800)
 	ammo_type = /obj/item/ammo_casing/a556
-	max_ammo = 10
+	max_ammo = 30

```

Comment:
Whatever the STS used to use was my intention for this, if it was 20, then the bullpup is (as far as I can tell) pretty much just an STS resprite. If it was originally 30. I'm not sure what exactly you meant there, Yosh.